### PR TITLE
CONSOLE-2472: Add initial quick starts

### PR DESCRIPTION
**Pull request by spadgett**
This PR contributes the initial out-of-the-box quick start custom resources and adds a brief README discussing the contribution process.

[CONSOLE-2472](https://issues.redhat.com/browse/CONSOLE-2472)

/assign @jhadvig

### CODE REVIEW

**Comment by spadgett** [comment]
cc @christianvogt @rohitkrai03 @alimobrem

File: quickstarts/README.md
Comment (christianvogt) [inline quickstarts/README.md:39]:
```suggestion
a Bugzilla bug. The `/cherry-pick` bot command will automatically create a
```

Comment (spadgett) [reply quickstarts/README.md]: Thanks, I pushed a fix for this typo

**Comment by spadgett** [comment]
/hold

The manifests are missing the cluster profile annotations.

**Comment by spadgett** [comment]
/hold cancel

Updated with the annotations.

**Comment by spadgett** [comment]
@beanh66 FYI

File: quickstarts/explore-pipeline-quickstart.yaml
Comment (christianvogt) [inline quickstarts/explore-pipeline-quickstart.yaml:16]:
```suggestion
```

File: quickstarts/explore-serverless-quickstart.yaml
Comment (christianvogt) [inline quickstarts/explore-serverless-quickstart.yaml:17]:
```suggestion
```

File: quickstarts/sample-application-quickstart.yaml
Comment (christianvogt) [inline quickstarts/sample-application-quickstart.yaml:16]:
```suggestion
```

File: quickstarts/add-healthchecks-quickstart.yaml
Comment (christianvogt) [inline quickstarts/add-healthchecks-quickstart.yaml:13]:
```suggestion
```

File: quickstarts/monitor-sampleapp-quickstart.yaml
Comment (christianvogt) [inline quickstarts/monitor-sampleapp-quickstart.yaml:13]:
```suggestion
```

File: quickstarts/monitor-sampleapp-quickstart.yaml
Comment (christianvogt) [inline quickstarts/monitor-sampleapp-quickstart.yaml:70]:
```suggestion
```

File: quickstarts/sample-application-quickstart.yaml
Comment (christianvogt) [inline quickstarts/sample-application-quickstart.yaml:13]:
```suggestion
```

**Review by christianvogt — COMMENTED** [review]
@spadgett I know this isn't your doing as the yaml files were provided. But we should remove the the empty properties.

File: quickstarts/sample-application-quickstart.yaml
Comment (jhadvig) [inline quickstarts/sample-application-quickstart.yaml:100]:
```suggestion
    quick start.
```

File: quickstarts/explore-serverless-quickstart.yaml
Comment (jhadvig) [inline quickstarts/explore-serverless-quickstart.yaml:14]:
```suggestion
  description: Install the OpenShift® Serverless Operator to deploy stateless, event-trigger-based
```

**Review by jhadvig — CHANGES_REQUESTED** [review]
Two small nits.
Also want to ask if we shall differentiate between `OpenShift®` and `OpenShift` cause the usage is a bit strange.

File: quickstarts/explore-serverless-quickstart.yaml
Comment (christianvogt) [reply quickstarts/explore-serverless-quickstart.yaml:14]: There are so many instances of `OpenShift` and a handful instances of `OpenShift®`.
I don't think it's worth getting into the semantics in this PR and content writers can revise them later.
I reached out to UX and their guidance was:
> We chatted about this with CCS and they reached out to legal who recommended marking the first appearance of trademarked terms

**Comment by spadgett** [comment]
Thanks @christianvogt and @jhadvig. I just pushed some the recommended updates.

File: quickstarts/explore-serverless-quickstart.yaml
Comment (beaumorley) [reply quickstarts/explore-serverless-quickstart.yaml:14]: We'll look at the copy when it is all migrated and merged. May have an update like we did before that we would want to do as z stream.

**Comment by christianvogt** [comment]
Thanks @spadgett any content changes should be a followup PR.

File: quickstarts/explore-serverless-quickstart.yaml
Comment (jhadvig) [reply quickstarts/explore-serverless-quickstart.yaml:14]: Makes sense, I was just a bit confused whats the right way how to use them.

**Comment by jhadvig** [comment]
@spadgett the PR looks good 👍 
One more thing what comes to my mind is if we want to add an e2e test in the console-operator, that would check if the QSs are created. But not sure if that makes sense, since the operator is not managing them. We can probably leave that to the future.

**Comment by spadgett** [comment]
> @spadgett the PR looks good 👍
> One more thing what comes to my mind is if we want to add an e2e test in the console-operator, that would check if the QSs are created. But not sure if that makes sense, since the operator is not managing them. We can probably leave that to the future.

Yeah, I was thinking about that. I wonder if this would be best as an e2e test in the console repo (since we run the console e2e tests here). Then we can confirm that they're showing up correctly in the UI.

**Pull request merged**
